package.json

```diff
@@ -18,47 +18,39 @@
   },
   "main": "lib/sth",
   "engines": {
-    "node": ">=4.8.4"
+    "node": ">=8.12.0"
   },
   "scripts": {
     "start": "node ./lib/sth",
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/unit/' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w ./lib",
+    "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch": "watch 'npm test && npm run lint' ./lib ./test"
   },
   "devDependencies": {
-    "chai": "~3.5.0",
+    "jshint": "~2.9.6",
     "clear-require": "~1.0.1",
     "expect.js": "~0.3.1",
-    "grunt": "~0.4.5",
-    "grunt-contrib-clean": "~1.0.0",
-    "grunt-contrib-jshint": "~1.1.0",
-    "grunt-contrib-watch": "~1.0.0",
-    "grunt-dox": "~0.5.0",
-    "grunt-exec": "~1.0.0",
-    "grunt-githooks": "~0.6.0",
-    "grunt-github-pages": "~0.0.4",
-    "grunt-gjslint": "~0.2.1",
-    "grunt-mkdir": "~1.0.0",
-    "grunt-mocha-test": "~0.13.2",
-    "grunt-plato": "~1.4.0",
     "istanbul": "~0.4.5",
-    "mocha": "~3.2.0",
+    "mocha": "5.2.0",
     "proxyquire": "1.7.10",
-    "request": "~2.79.0",
-    "sinon": "~1.17.6",
-    "sinon-chai": "2.8.0"
+    "request": "2.88.0",
+    "watch": "~1.0.2"
   },
   "dependencies": {
     "app-root-path": "~1.2.1",
     "async": "~2.0.0-rc.5",
-    "boom": "~2.7.2",
+    "boom": "7.2.2",
     "bytes-counter": "~1.0.0",
     "commander": "~2.9.0",
     "csv-parser": "~1.9.3",
-    "hapi": "~16.1.1",
-    "joi": "~5.1.0",
-    "json-csv": "~1.2.0",
+    "hapi": "16.7.0",
```

Comment:
`npm audit` suggested veresion 17.7.0. However, this required code adaptations too hard for the time being (see details at issue https://github.com/telefonicaid/fiware-sth-comet/issues/468) so I opted for a minor in 16 that also have the vulnerability solved.

NTC (just informative)